README.md

```diff
@@ -1,14 +1,37 @@
-# solid-namespace
+# Solid Namespace
 [![](https://img.shields.io/badge/project-Solid-7C4DFF.svg?style=flat)](https://github.com/solid/solid)
 [![NPM Version](https://img.shields.io/npm/v/solid-namespace.svg?style=flat)](https://npm.im/solid-namespace)
 
 A collection of common RDF namespaces used in the Solid project.
 
+solid-namespace can be used with any RDF/JS-compatible library (e.g. [rdflib.js](/linkeddata/rdflib.js)). 
+
+
 ## Usage
 
+You can use this library in two ways. 
+
+1. With a RDF JS library to get NamedNodes
+2. Without a library to get url strings
```

Comment:
Problem is that 1. now works with a function, but 2. as an object. My solution makes both a function.